static/src/javascripts/projects/commercial/modules/messenger/click.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { trackNativeAdLinkClick } from 'common/modules/analytics/google';
+import { register } from 'commercial/modules/messenger';
+
+const sendClick = (adSlot: Element, linkName: string): void => {
+    trackNativeAdLinkClick(adSlot.id, linkName);
+};
+
+register('click', (linkName, ret, iframe = {}) =>
+    sendClick(
+        iframe.closest('.js-ad-slot') || {
+            id: 'unknown',
+        },
+        linkName || ''
+    )
+);
+
+export { sendClick };
```

Comment:
Hmm. I prefer to add the `export` statement to the function declaration, however, then I run into the `Export statements should appear at the end of the file` error

If I move `export const sendClick = ` to the end of the file, then I get a `no-use-before-define` error within register().

I'm currently uncertain if I prefer adding a ES Lint disable comment, or doing a separate export. Maybe leaning slightly towards the latter. If anyone has strong convictions, I'm happy to change it 😁